app/controllers/agentAppointment/AddAgentController.scala

```diff
@@ -327,7 +338,7 @@ class AddAgentController @Inject()(
     authenticated.andThen(withAppointAgentSession).async { implicit request =>
       val backLink =
         if (fromCyaChange) routes.CheckYourAnswersController.onPageLoad().url
-        else routes.AddAgentController.isCorrectAgent.url
+        else routes.RatingListOptionsController.show().url
```

Comment:
I think this will 404 if the flag is disabled and the user goes back from 'Which of your properties do you want to assign test company name to?'